plugins/source/datadog/docs/tables/datadog_dashboard_lists.md

```diff
@@ -0,0 +1,23 @@
+# Table: datadog_dashboard_lists
+
+
+
+The primary key for this table is **id**.
+
+
+## Columns
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id|UUID|
+|_cq_parent_id|UUID|
+|account_name|String|
+|id (PK)|Int|
```

Comment:
Is `id` guaranteed to be unique across accounts, or should `account_name` be part of the PK here? (same question for other tables) It's probably safer to include `account_name` in the PK, so we don't risk overwriting rows of other accounts?